components/places/src/places.udl

```diff
@@ -330,12 +330,6 @@ enum PlacesError {
     "InternalPanic",
 };
 
-enum BookmarkType {
```

Comment:
I removed this because it's no longer being used as a-c has defined its own `BookmarkNodeType`.